DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/SetDelegatePassword/Index.cshtml

```diff
@@ -41,7 +42,7 @@
                      populate-with-current-value="false"
                      type="text"
                      spell-check="false"
-                     autocomplete=""
+                     autocomplete="off"

```

Comment:
In other places we use `new-password` instead of off. Is there a reason not to use that here?

Reply:
I didn't know about that option, but I've just tried changing to that and it doesn't work. The autocomplete with the standard admin password shows.

Reply:
Nope, all 3 of those fields work as expected, but all are password inputs not just regular inputs.